rust-toolchain

```diff
@@ -1 +1 @@
-1.40.0
```

Comment:
We should note that version bump in our changelog. Just update the new minimal required rust version there.

Reply:
I understand. Then, should I add an explanation somewhere that we should use the rust version over 1.48.0 in order to build docs locally?

Reply:
#2657 will also bump generally bump the minimal supported rust version to 1.48, so it's fine to update this just now and adjust the corresponding changelog entry.

Reply:
Done with it!